indico/modules/categories/models/event_move_request.py

```diff
@@ -54,6 +54,10 @@ class EventMoveRequest(db.Model):
         default=MoveRequestState.pending,
         nullable=False
     )
+    state_reason = db.Column(
+        db.String,
+        nullable=True
```

Comment:
I'd use `nullable=False` and `default=''`; otherwise we'll almost certainly end up with a mix of empty string and NULLs at some point...

And maybe "moderator_comment" would be a better name? "state_reason" sounds a bit strange to me..

Reply:
Changed in #5013